newrelic/dashboard/dashboard.tf

```diff
@@ -0,0 +1,618 @@
+resource "newrelic_dashboard" "tfer--license-002D-usage-002D-12-002D-months-002D-825238" {
```

Comment:
I worry the usability tradeoff of doing/managing/syncing dashboard configuration through Terraform (rather than the UI) isn't worth it, since we aren't trying to automate setup of a bunch of identical dashboards or anything. Thoughts?

Not going to dig through the specifics of the dashboard here; can work through that separately.

Reply:
Same as alerts, the dashboards are all created in the UI terraform just saves these as ID to ensure they aren't removed once created.